### PR TITLE
Add harvester load balancer chart

### PR DESCRIPTION
**Pull request by yaocw2020**
issue: https://github.com/harvester/harvester/issues/1019

### CODE REVIEW

**Comment by yaocw2020** [comment]
It should be merged after the version 0.1.1 release.

File: charts/harvester-load-balancer/values.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: rancher/harvester-load-balancer
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```
Comment (guangbochen) [inline charts/harvester-load-balancer/values.yaml:11]: set default to master-head as this is dev version

**Review by guangbochen — COMMENTED** [review]
will need to fix the CI

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**